sriovnet_switchdev.go

```diff
@@ -226,6 +226,44 @@ func findNetdevWithPortNameCriteria(criteria func(string) bool) (string, error)
 	return "", fmt.Errorf("no representor matched criteria")
 }
 
+// Finds the VF index of a port, if applicable
+func GetVfIndexFromNetDevice(netdev string) (int, error) {
```

Comment:
nit: GetVfIndexFromRepresentor(repNetDev string) (int, error) ?

also please update comment to be according to go practices. (added comment above)